package.json

```diff
@@ -15,18 +15,21 @@
   },
   "scripts": {
     "dev:chrome": "cross-env NODE_ENV=development cross-env TARGET_BROWSER=chrome webpack --watch",
+    "build:style": "postcss src/styles/tailwind.css -o src/styles/index.css",
     "dev:firefox": "cross-env NODE_ENV=development cross-env TARGET_BROWSER=firefox webpack --watch",
     "dev:opera": "cross-env NODE_ENV=development cross-env TARGET_BROWSER=opera webpack --watch",
     "build:chrome": "cross-env NODE_ENV=production cross-env TARGET_BROWSER=chrome webpack",
     "build:firefox": "cross-env NODE_ENV=production cross-env TARGET_BROWSER=firefox webpack",
     "build:opera": "cross-env NODE_ENV=production cross-env TARGET_BROWSER=opera webpack",
-    "build": "yarn run build:chrome && yarn run build:firefox && yarn run build:opera",
+    "build": "craco build && npm run build:style && yarn run build:chrome && yarn run build:firefox && yarn run build:opera",
```

Comment:
why do we need that?
can this be part of webpack? 
and if we need it we probably also need it in the other `build` tasks?

Reply:
I made a change for it. It generates the tailwind css file, we don't need it in every build, just once or when adding new things

Reply:
ah ok, thanks.

and the `craco build` ? is this also only needed for the styles? (if so, can we add it to the `build:style` command?)